worlds/rabi_ribi/__init__.py

```diff
@@ -145,3 +219,9 @@ def _handle_encourage_eggs_in_late_spheres(multiworld):
     @classmethod
     def stage_post_fill(cls, multiworld) -> None:
         cls._handle_encourage_eggs_in_late_spheres(multiworld)
+
+    # For the universal tracker, doesn't get called in standard gen
+    @staticmethod
+    def interpret_slot_data(slot_data: Dict[str, Any]) -> Dict[str, Any]:
+        # Returning slot_data so it regens, giving it back in multiworld.re_gen_passthrough
+        return slot_data
```

Comment:
Similar confusion about why this is needed. Can we describe the motivation behind this method in a docstring?

Reply:
Updated the documentation here + added a comment with a link to the Universal Tracker documentation.